### PR TITLE
Added 20NG dataset

### PR DESCRIPTION
**Pull request by jmysliv**

### CODE REVIEW

File: bin/data_preprocessing/tng.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+import numpy as np
+
+
+def get_data(size):
+    tng = pd.read_csv('data/tng.csv', names=([i for i in range(5000)] + ['labels']))
```
Comment (MikeyZat) [inline bin/data_preprocessing/tng.py:6]:
```suggestion
    tng = pd.read_csv('data/tng.csv', names=([i for i in range(size)] + ['labels']))
```

**Review by MikeyZat — APPROVED** [review]
Just one small change

File: bin/data_preprocessing/tng.py (hunk 1 above)
Comment (jmysliv) [reply bin/data_preprocessing/tng.py:6]: `5000` is the number of features. And `size` in this context is the number of rows.

Comment (MikeyZat) [reply bin/data_preprocessing/tng.py:6]: Okay, nvm then

**Pull request merged**